README.md

```diff
@@ -103,10 +103,13 @@ networkInterface.use([{
     request.variables = {name: 'Luke'}
     
     // Or Just add authorization token
-    request.context = {authorization: 'jwt_token'}
+    options.params = {
+      ...options.params,
+      guardian_token: 'my-secret-token',
```

Comment:
As not everyone uses Guardian (and the app shouldn't really know about that anyway), I'd suggest a more neutral name for the token here 😊 How about `auth_token`?

Reply:
I agree with you, done!